Introduction to xemc3.ipynb

```diff
@@ -0,0 +1,3171 @@
+       "         [5.90682402, 5.90128975, 5.89034014, ..., 4.27331521,\n",
+       "          4.21872462, 4.16492863]],\n",
+       " \n",
+       "        [[5.88039947, 5.87478717, 5.86353702, ..., 4.22636359,\n",
+       "          4.17204559, 4.11853643],\n",
+       "         [5.88048206, 5.87503477, 5.86394578, ..., 4.22725027,\n",
+       "          4.17290028, 4.11936057],\n",
+       "         [5.88056438, 5.87528231, 5.86435491, ..., 4.22815464,\n",
+       "          4.17377244, 4.12020195],\n",
+       " ...\n",
+       "         [5.64755073, 5.63505485, 5.61354696, ..., 3.81421366,\n",
+       "          3.76543708, 3.71779086],\n",
+       "         [5.64922161, 5.63819013, 5.61807245, ..., 3.81753395,\n",
+       "          3.76870465, 3.7210289 ],\n",
+       "         [5.65069075, 5.64113616, 5.6224387 , ..., 3.82111107,\n",
+       "          3.77222496, 3.7245137 ]],\n",
+       " \n",
+       "        [[5.59414493, 5.58958172, 5.57387846, ..., 3.75650956,\n",
+       "          3.70552132, 3.65647075],\n",
+       "         [5.59422224, 5.59117652, 5.57709579, ..., 3.76073063,\n",
+       "          3.70983664, 3.66088151],\n",
+       "         [5.59408138, 5.5925006 , 5.5799317 , ..., 3.76497918,\n",
+       "          3.71417615, 3.66525302],\n",
+       "         ...,\n",
+       "         [5.59027424, 5.58100907, 5.56053413, ..., 3.7435559 ,\n",
+       "          3.69303175, 3.64369176],\n",
+       "         [5.59192682, 5.58422348, 5.56529866, ..., 3.74788951,\n",
+       "          3.69714638, 3.64790418],\n",
+       "         [5.59337562, 5.58724114, 5.56988255, ..., 3.75223099,\n",
+       "          3.701277  , 3.65213093]]])\n",
+       " Dimensions without coordinates: r, theta, phi,\n",
+       " <xarray.DataArray 'Te' (r: 139, theta: 512, phi: 36)>\n",
+       " [2562048 values with dtype=float64]\n",
+       " Dimensions without coordinates: r, theta, phi\n",
+       " Attributes:\n",
+       "     long_name:  Electron temperature\n",
+       "     units:      eV)"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "R = ds.R_bounds.mean(dim=(\"delta_r\", \"delta_theta\", \"delta_phi\"))\n",
+    "z = ds.z_bounds.mean(dim=(\"delta_r\", \"delta_theta\", \"delta_phi\"))\n",
+    "phi = ds.phi_bounds.mean(dim=\"delta_phi\")\n",
+    "x = R * np.cos(phi)\n",
+    "y = R * np.sin(phi)\n",
+    "x, ds.Te"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 125,
+   "id": "b2b57e7d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total number of gridpoints in the W7X gridstructure is  2562048\n"
+     ]
+    }
+   ],
+   "source": [
+    "x.shape\n",
+    "tot_grid_points = x.shape[0]*x.shape[1]*x.shape[2]\n",
+    "print(\"Total number of gridpoints in the W7X gridstructure is \", tot_grid_points)"
```

Comment:
that is the number of cells, not grid points.
Also `x.size` gives the same info